<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-SDVCEDF4" failures="5" tests="10" name="Test" time="5.882" errors="0" timestamp="2023-02-20T21:14:16 ICT">
  <testcase classname="com.api.auto.testcase.TC_API_Login" name="TC01_Validate200Ok" time="0.015"/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork" name="TC01_Validate201Created" time="0.004">
    <failure type="java.lang.AssertionError" message="Status Check Failed! expected [201] but found [404]">
      <![CDATA[java.lang.AssertionError: Status Check Failed! expected [201] but found [404]
at com.api.auto.testcase.TC_API_CreateWork.TC01_Validate201Created(TC_API_CreateWork.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- TC01_Validate201Created -->
  <testcase classname="com.api.auto.testcase.TC_API_Login" name="TC02_ValidateMessage" time="0.936"/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork" name="TC02_ValidateWorkId" time="0.002">
    <failure type="java.lang.AssertionError" message="Respone body không chứa trường id! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Respone body không chứa trường id! expected [true] but found [false]
at com.api.auto.testcase.TC_API_CreateWork.TC02_ValidateWorkId(TC_API_CreateWork.java:59)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- TC02_ValidateWorkId -->
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork" name="TC03_ValidateNameOfWorkMatched" time="0.002">
    <failure type="java.lang.AssertionError" message="Respone body không chứa trường nameWork! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Respone body không chứa trường nameWork! expected [true] but found [false]
at com.api.auto.testcase.TC_API_CreateWork.TC03_ValidateNameOfWorkMatched(TC_API_CreateWork.java:64)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- TC03_ValidateNameOfWorkMatched -->
  <testcase classname="com.api.auto.testcase.TC_API_Login" name="TC03_ValidateToken" time="0.037"/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork" name="TC03_ValidateExperienceMatched" time="0.002">
    <failure type="java.lang.AssertionError" message="Respone body không chứa trường experience! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Respone body không chứa trường experience! expected [true] but found [false]
at com.api.auto.testcase.TC_API_CreateWork.TC03_ValidateExperienceMatched(TC_API_CreateWork.java:72)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- TC03_ValidateExperienceMatched -->
  <testcase classname="com.api.auto.testcase.TC_API_Login" name="TC05_ValidateUserType" time="0.019"/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork" name="TC03_ValidateEducationMatched" time="0.002">
    <failure type="java.lang.AssertionError" message="Respone body không chứa trường education! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Respone body không chứa trường education! expected [true] but found [false]
at com.api.auto.testcase.TC_API_CreateWork.TC03_ValidateEducationMatched(TC_API_CreateWork.java:80)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- TC03_ValidateEducationMatched -->
  <testcase classname="com.api.auto.testcase.TC_API_Login" name="TC06_ValidateAccount" time="0.033"/>
</testsuite> <!-- Test -->
